packages/mdc-ripple/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@material/base": "^0.35.0",
-    "@material/theme": "^0.35.0"
+    "@material/theme": "^0.35.0",
+    "@material/animation": "^0.35.0"
```

Comment:
This should be `0.34.0`. We haven't released `0.35.0` of animation yet.
https://github.com/material-components/material-components-web/blob/e6b787c4dfdd0b6810534d929b3aad2011ab19d3/packages/mdc-animation/package.json#L4

Reply:
That's why the CI tests were failing, too.

Reply:
Done. Thanks!